Tavisca.Bootcamp.LanguageBasics.Exercise1/Program.cs

```diff
@@ -13,17 +13,54 @@ static void Main(string[] args)
             Test("2*12?=247", -1);
             Console.ReadKey(true);
         }
-
-        private static void Test(string args, int expected)
-        {
+        
+        private static void Test(string args, int expected)
+        {
             var result = FindDigit(args).Equals(expected) ? "PASS" : "FAIL";
-            Console.WriteLine($"{args} : {result}");
+            Console.WriteLine($"{args} : {result}");
         }
 
-        public static int FindDigit(string equation)
-        {
+        public static int FindDigit(string equation)

```

Comment:
Move this method to another class, as mentioned in Problem statement

Reply:
Changes done .
@tavisca-njain